src/screens/Welcome.tsx

```diff
@@ -1,25 +1,28 @@
-import React, { useState, useEffect } from "react";
-import { Button, ContextSelectorFooter } from "@patternfly/react-core";
+import React, { useEffect } from "react";
+import { Button } from "@patternfly/react-core";
 import { useNavigate } from 'react-router-dom';
 
 function Welcome() {
   const navigate = useNavigate();
-  const [user, setUser] = useState();
+  // const [user, setUser] = useState();
   useEffect(() => {
     const loggedInUser = localStorage.getItem("user");
     if (loggedInUser) {
       const foundUser = JSON.parse(loggedInUser);
       // Don't set user if it is undefined
-      if (foundUser != undefined) {
-        setUser(foundUser);
+      if (foundUser !== undefined) {
+        // setUser(foundUser);
         console.log("User is logged in");
         console.log(foundUser.email);
         navigate("/home");
       }else{
         console.log("User not logged in");
       }
```

Comment:
We should also verify that their credentials are valid. I would assume Firebase provides a way of doing this automatically so let's check into that. In addition, Firebase SDK might have some sort of hook etc so we don't have to manually check for stored credentials in the cache etc. Please look into this and report back with what you find!

Reply:
https://firebase.google.com/docs/reference/js/v8/firebase.auth.Auth

Check out the `onAuthStateChanged` function. I think we can leverage this. Check out the useOnAuthState section in this tutorial: https://designcode.io/react-hooks-handbook-firebase-auth

Reply:
@christorange Jason is working on this I think? So no need to include on this PR but make sure it's on Trello.